shell/store/index.js

```diff
@@ -157,25 +159,28 @@ const updateActiveNamespaceCache = (state, activeNamespaceCache) => {
 
 export const state = () => {
   return {
-    managementReady:         false,
-    clusterReady:            false,
-    isMultiCluster:          false,
-    isRancher:               false,
-    namespaceFilters:        [],
-    activeNamespaceCache:    {}, // Used to efficiently check if a resource should be displayed
-    activeNamespaceCacheKey: '', // Fingerprint of activeNamespaceCache
-    allNamespaces:           [],
-    allWorkspaces:           [],
-    clusterId:               null,
-    productId:               null,
-    workspace:               null,
-    error:                   null,
-    cameFromError:           false,
-    pageActions:             [],
-    serverVersion:           null,
-    systemNamespaces:        [],
-    isSingleProduct:         undefined,
-    namespaceFilterMode:     null,
+    managementReady:           false,
+    clusterReady:              false,
+    isMultiCluster:            false,
+    isRancher:                 false,
+    namespaceFilters:          [],
+    activeNamespaceCache:      {}, // Used to efficiently check if a resource should be displayed
+    activeNamespaceCacheKey:   '', // Fingerprint of activeNamespaceCache
+    allNamespaces:             [],
+    allWorkspaces:             [],
+    clusterId:                 null,
+    productId:                 null,
+    workspace:                 null,
+    error:                     null,
+    cameFromError:             false,
+    pageActions:               [],
+    serverVersion:             null,
+    systemNamespaces:          [],
+    isSingleProduct:           undefined,
+    namespaceFilterMode:       null,
+    supportRancherManage:      true,
+    openRancherManagerSupport: false, // Use the harvester embedded rancher
```

Comment:
This is a little nitpicky of me but it's not super clear what this variable is for in some of the other places its used and might be confusing for devs not familiar with the Harvester feature set. Something with Harvester in the name like `isRancherInHarvester`  would be clearer. Also it looks like `supportRancherManage` is not being used.